frontend/src/_styles/theme.scss

```diff
@@ -5396,7 +5396,7 @@ div#driver-page-overlay {
 .workspace-settings-page {
   width: 880px;
   margin: 0 auto;
-  background: var(--base);
+  // background: var(--base);
```

Comment:
Shall we remove the commented line?

Reply:
Yes, that would be fine. Do you want to make the change?

Reply:
Yes @IWang20